app/routes/groups.tsx

```diff
@@ -1,15 +1,67 @@
-import { useGroups } from '~/hooks/useGroups';
-import { Link } from '@remix-run/react';
+import { Form, Link, useLoaderData } from '@remix-run/react';
 import { Image } from '~/components/ui/images';
 import { LinkButton } from '~/components/ui/forms';
+import { db } from '~/modules/database/db.server';
+import { type LoaderFunctionArgs } from '@remix-run/node';
+import type { Group } from '@prisma/client';
```

Comment:
NIT: Let's keep the `type` syntax consistent when we can. Maybe we can figure out how to add a lint rule for this in a future Eslint session :)